src/js/out/modules/thirdparty/ws.js

```diff
@@ -59,51 +66,83 @@ class BunWebSocket extends EventEmitter {
   #paused = !1;
   #fragments = !1;
   #binaryType = "nodebuffer";
-  readyState = BunWebSocket.CONNECTING;
```

Comment:
`readyState` is an expected attribute here. https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/readyState

Reply:
Though we could make it a getter which just returns from the underlying websocket.

Reply:
That's what I did